MaxText/kernels/megablox/gmm.py

```diff
@@ -400,8 +411,8 @@ def gmm(
   def kernel(
```

Comment:
The "we need to know contracting axis when we quantized lhs and rhs" section above this line is also irrelevant any more.